src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "username";
+    private static final String VALID_PASSWORD = "123456";
+    private static final int VALID_AGE = 18;
+    private static final String INVALID_LOGIN = "name";
+    private static final String INVALID_PASSWORD = "1234";
+    private static final int INVALID_AGE = 16;
+    private static final String EMPTY_VALUE = "";
```

Comment:
it's the same as INVALID_PASSWORD in your tests, cause both make test check your code's behavior with string.length() < 6